secure_message/repository/retriever.py

```diff
@@ -217,11 +217,11 @@ def _retrieve_thread_for_internal_user(thread_id):
                 .order_by(Status.id.desc())
 
             if not result.all():
-                logger.debug('Thread does not exist', thread_id=thread_id)
-                raise NotFound(description=f"Conversation with thread_id '{thread_id}' does not exist")
+                logger.error('Thread not retrieved for internal user', thread_id=thread_id)
+                raise NotFound(description=f"Conversation with thread_id '{thread_id}' not retrieved")
 
         except SQLAlchemyError:
-            logger.exception('Error retrieving conversation from database')
+            logger.exception(f"Error retrieving conversation from database thread_id '{thread_id}'")
```

Comment:
Use fields instead of fstrings.  Makes it easier to search for in splunk

Reply:
Same above